test/metabase/query_processor/streaming/xlsx_test.clj

```diff
@@ -643,7 +643,7 @@
 (deftest encode-strange-classes-test
   (testing (str "Make sure that we're piggybacking off of the JSON encoding logic when encoding strange values in "
                 "XLSX (#5145, #5220, #5459)")
-    (is (= ["Hello XLSX World!" "{:v \"No Encoder\"}"]
+    (is (= ["\"Hello XLSX World!\"" "{\"v\":\"No Encoder\"}"]
```

Comment:
Devil's advocate but I think the old unquoted string behavior was probably preferrable. Someone is probably going to open a regression if we start quoting string column values in XLSX downloads when we didn't before. Maybe it's worth having a method specifically for `String`.